prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -215,7 +215,8 @@ function getLastRCVersion() {
 function installKyma() {
   kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value account)"
   mkdir -p /tmp/kyma-gke-upgradeability
-  LAST_RELEASE_VERSION=$(kyma::get_last_release_version "${BOT_GITHUB_TOKEN}")
+  kyma::get_last_release_version "${BOT_GITHUB_TOKEN}"
```

Comment:
Please rewrite kyma::get_last_release_version to accept short params instead positional.